municipal_finance/static/stylesheets/scorecard.scss

```diff
@@ -752,10 +752,42 @@ a {
   }
 
   // icomoon icons
-  .rating-great:after { content: "\e9e0\0020\e99e"; }
   .rating-good:after { content: "\e9e2"; }
   .rating-ave:after { content: "\e9f8"; }
   .rating-bad:after { content: "\e9e6"; }
+
+  .audit-rating {
+    font-family: 'fontawesome' !important;
+    speak: none;
+    font-style: normal;
+    font-weight: normal;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+    margin-left: 5px;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+
+  .audit-unqualified,
+  .audit-unqualified_emphasis_of_matter { color: $green; }
+  .audit-unqualified_extra {
+    color: $yellow;
+    margin-left: 0px;
+  }
+  .audit-qualified,
+  .audit-adverse,
+  .audit-disclaimer { color: $red; }
+
+  .audit-unqualified:after { content: "\f164"; }
+  .audit-unqualified_emphasis_of_matter:after { content: "\f164"; }
```

Comment:
I thought the same. Ofc that means no class needed
On 06 Jun 2016 13:30, "Greg Kempe" <notifications@github.com> wrote:

> In municipal_finance/static/stylesheets/scorecard.scss
> <https://github.com/Code4SA/municipal-data/pull/163#discussion_r65875008>:
>
> > +    -webkit-font-smoothing: antialiased;
> > +    -moz-osx-font-smoothing: grayscale;
> > +  }
> > +
> > +  .audit-unqualified,
> > +  .audit-unqualified_emphasis_of_matter { color: $green; }
> > +  .audit-unqualified_extra {
> > +    color: $yellow;
> > +    margin-left: 0px;
> > +  }
> > +  .audit-qualified,
> > +  .audit-adverse,
> > +  .audit-disclaimer { color: $red; }
> > +
> > +  .audit-unqualified:after { content: "\f164"; }
> > +  .audit-unqualified_emphasis_of_matter:after { content: "\f164"; }
>
> Nishendra said to treat it neutrally. I decided that meant no icon.
> Thoughts? Greg -- (iPhone)
> … <#m_-3116669619318496334_>
> On 06 Jun 2016, at 12:35 PM, JD Bothma ***@***.***> wrote: In
> municipal_finance/static/stylesheets/scorecard.scss: > +
> -webkit-font-smoothing: antialiased; > + -moz-osx-font-smoothing:
> grayscale; > + } > + > + .audit-unqualified, > +
> .audit-unqualified_emphasis_of_matter { color: $green; } > +
> .audit-unqualified_extra { > + color: $yellow; > + margin-left: 0px; > + }
> > + .audit-qualified, > + .audit-adverse, > + .audit-disclaimer { color:
> $red; } > + > + .audit-unqualified:after { content: "\f164"; } > +
> .audit-unqualified_emphasis_of_matter:after { content: "\f164"; } missing
> outstanding? — You are receiving this because you modified the open/close
> state. Reply to this email directly, view it on GitHub, or mute the thread.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Code4SA/municipal-data/pull/163/files/79c4a917a07357b028a7864f335b1f1a0e948b11#r65875008>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAOZGSH5bX9ltXcUtR-NfoXjVQI0YKWYks5qJATWgaJpZM4Iuw5G>
> .
>